### DISCUSSION TITLE
Setting dictionary cover icon for EPWING format dictionaries?

### DISCUSSION THREAD

**Discussion by midnitetoker42** [xiaoyifang/goldendict-ng#1530] [Q&A] [answered]
Just started using the program recently. I loaded a number of dictionaries, most in MDX format, some in EPWING format. On the goldendict-ng toolbar, there are icons to select which dictionaries to include when searching. Those icons are using images from the dictionary folders to represent the dictionary. For example, a folder contains "大辞泉.mdd" and "大辞泉.jpg". The JPG file is used as the icon for the dictionary. However, I can't seem to figure out how to do the same for an EPWING format dictionary. I tried placing a jpg named the same as the folder name in the parent folder and in the folder itself, I tried a CATALOGS.jpg, a HONMON.jpg in the folders where those source files are but they aren't used. The dictionaries get the default icon instead. Screenshot attached for clarity.

Is there a way for me to change the cover/icon image for a dictionary that is in EPWING format? How? Thanks for a great program otherwise!
![goldendict-covers](https://github.com/xiaoyifang/goldendict-ng/assets/93347639/321190a8-a98c-4adc-ac7c-82d50d44f1a5)

**Comment by shenlebantongying** [comment, marked as answer]
It is supposed to work as described here which is copied without change from original GD's help.

https://xiaoyifang.github.io/goldendict-ng/dictformats/#individual-dictionary-icons

I cannot try now, but it says

> For Epwing dictionaries icon file must be named by name of folder with dictionary data beside "catalogs" file

**Comment by midnitetoker42** [comment]
Fantastic, worked perfectly! Thanks for the quick & accurate answer.
![image](https://github.com/xiaoyifang/goldendict-ng/assets/93347639/5ee060a3-56c4-4dca-8cf3-00494da2276b)
